engine/CMakeLists.txt

```diff
@@ -32,6 +32,10 @@
 
 MESSAGE("PROCESSING DIRECTORY: ${CMAKE_CURRENT_SOURCE_DIR}")
 
+CMAKE_MINIMUM_REQUIRED(VERSION 3.21 FATAL_ERROR)
+
+SET(X_VCPKG_APPLOCAL_DEPS_INSTALL ON)
```

Comment:
based on https://github.com/microsoft/vcpkg/blob/d7112d1a4fb50410d3639f5f586972591d848beb/scripts/buildsystems/vcpkg.cmake#L52 this would be target dependent; though not entirely sure we should depend on it given its an experimental option for VCPKG.

Reply:
That's fine - let's just make sure we monitor the status of it in case it changes one way or the other.